lib/endpoints/class-wp-json-attachments-controller.php

```diff
@@ -167,6 +167,9 @@ public function prepare_item_for_response( $post, $request ) {
 		    $response['media_details']['sizes'] = new stdClass;
 		}
 
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
```

Comment:
> can this be handled by a default param in the args in the route registration?

Maybe. But, conceivably, another controller can call this directly and omit `context`